articles/cosmos-db/consistency-levels.md

```diff
@@ -66,23 +66,28 @@ Strong consistency offers a linearizability guarantee. Linearizability refers to
 
-Bounded staleness offers total global order outside of the "staleness window." When a client performs read operations within a region that accepts writes, the guarantees provided by bounded staleness consistency are identical to those guarantees by the strong consistency. As the staleness window approaches for either time or updates, whichever is closer, the service will throttle new writes to allow replication to catch up and honor the consistency guarantee.
+For a single-region account, Bounded Staleness provides the same write consistency guarantees as Session and Eventual Consistency with data being replicated to a local majority (three replicas in a four replica set) in the single region. 
 
-Inside the staleness window, Bounded Staleness provides the following consistency guarantees:
+> [!IMPORTANT]
+> With Bounded Staleness consistency, staleness checks are made only across regions and not within a region. Within a given region, data is always replicated to a local majority (three replicas in a four replica set) regardless of the consistency level.
 
-- Consistency for clients in the same region for an account with single write region = [Strong](#strong-consistency)
-- Consistency for clients in different regions for an account with single write region = [Consistent Prefix](#consistent-prefix-consistency)
-- Consistency for clients writing to a single region for an account with multiple write regions = [Consistent Prefix](#consistent-prefix-consistency)
-- Consistency for clients writing to different regions for an account with multiple write regions = [Eventual](#eventual-consistency)
+Reads when using Bounded Staleness will return the latest data available in that region by reading from two available replicas in that region. Since writes within a region always replicate to a local majority (3 out of 4 replicas), consulting two replicas will return the most up to date data available in that region.
+
+> [!IMPORTANT]
+> With Bounded Staleness consistency, reads issued against a non-primary region may not necessarily return the most recent version of the data globally, but are guaranteed to return the most recent version of the data in that region, which will be within the maximum staleness boundary globally.
+
+Bounded Staleness works best for globally distributed applications using multi-region, single-region write accounts, where near strong consistency across regions is desired. For multi-region, multi-region write accounts, application servers should direct reads and writes to the same region in which the application servers are hosted. Thus, Bounded Staleness in a multi-write account is an anti-pattern as it would require a dependency on replication lag between regions, which should not matter if data is read from the same region it was written to.
```

Comment:
@abinav2307, I'm confused a little on this statement as anti-pattern. Wouldn't Bounded Staleness still be useful in scenarios where there are multiple SDK instances in the same region if a user wasn't using the session token?

Reply:
That is true for single-region write accounts with two or more regions but there's a lot more in a multi-write setup with conflict resolution etc. that has the potential to bring in a dependency on replication lag when using Bounded Staleness. To avoid any dependency on replication lag when using a multi-write setup, Bounded Staleness should be avoided as much as possible.